packages/app-page-builder/src/editor/plugins/elementSettings/save/SaveDialog/graphql.ts

```diff
@@ -33,3 +33,14 @@ export const CREATE_FILE = gql`
         }
     }
 `;
+
```

Comment:
Please add a TODO here for the core team, to export these queries from the `app-file-manager` package, to make them reusable. Thanks!